src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -188,12 +182,6 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
                     .body(null);
         }
 
-        // Check if course shortname is set
-        if (course.getShortName() == null || course.getShortName().length() < 3) {
```

Comment:
Not needed since we fetch the course from the database a couple of lines before this check

Reply:
The short name could still be empty or too short as we don’t have a database constraint here, I guess this check does not harm!

Reply:
I reverted the change